README.md

```diff
@@ -282,43 +282,40 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
+                api("org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutinesVersion}")
                 implementation("com.juul.kable:core:${kableVersion}")
             }
         }
 
-        val androidMain by getting {
+        val macosX64Main by getting {
             dependencies {
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutinesVersion}")
+                api("org.jetbrains.kotlinx:kotlinx-coroutines-core-macosx64:${coroutinesVersion}-native-mt") {
```

Comment:
What's the need behind exposing this library from the iOS/Mac Native modules?

Reply:
`api` is needed if any of Kable (that exposes Coroutines classes) is used on the consuming side.

For example, if someone is writing an app that uses Kable and is being written in Kotlin and targeting iOS then they'll need Coroutines on their classpath (e.g. for `Flow` availability). I imagine that most people will define Coroutines in their app's `build.gradle.kts`, but defining as `api` ensures that it will work for them even if they don't define Coroutines in their app.

Reply:
Oh, I think I understand what you were getting at.
The `commonMain` already exposes the core APIs for Coroutines, so other specialized dependencies don't need to be `api`.
I'll update that. Good catch.